config/routes.rb

```diff
@@ -1,34 +1,37 @@
 Rails.application.routes.draw do
-  devise_for :users, controllers: { registrations: "registrations" }
-
   root to: 'pages#home'
+  get '/landing', to: 'pages#landing'
   get '/calendar', to: 'pages#calendar'
+  get '/files', to: 'pages#files'
+
+  devise_for :users, controllers: { registrations: "registrations" }
+  resources :users
+
+  get '/onboarding', to: 'users#onboarding', as: 'onboarding'
+  patch '/contract_update', to: 'users#contract_update', as: 'contract_update'
+  get '/personal_details', to: 'users#personal_details', as: 'personal_details'
+  patch '/personal_update', to: 'users#personal_update', as: 'personal_update'
 
-  get '/accidents/confirm', to: 'accidents#confirm'
-  post '/accidents/send', to: 'accidents#send_accident'
-  post '/workdays/send', to: 'workdays#send_worklog'
-  post '/payslips/send', to: 'payslips#send_payslip'
-  post '/users/send', to: 'users#send_ipa'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :accidents do
     member do
       get :new3
       get :complete
     end
   end
+  get '/accidents/confirm', to: 'accidents#confirm'
```

Comment:
I think this is the one that causes the confirm to not work

Reply:
Ohh ic, thanks for finding the specific one!